jvb/src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -200,6 +201,7 @@ public Videobridge(
             xmppConnection.setEventHandler(new XmppConnectionEventHandler());
         }
         this.version = version;
+        this.releaseId = "346"; /* $ Get real value from somewhere. */
```

Comment:
This should read the value from the config file (see here for an example of reading a nullable value https://github.com/jitsi/jitsi-videobridge/blob/master/jvb/src/main/kotlin/org/jitsi/videobridge/octo/config/OctoConfig.kt#L57)